OWNERS

```diff
@@ -1,14 +1,19 @@
 # approval == this is a good idea /approve
 approvers:
-  - russellb
   - dhellmann
-  - mhrivnak
   - hardys
+  - honza
+  - mhrivnak
+  - russellb
+  - stbenjam
   - zaneb
 # review == this code is good /lgtm
 reviewers:
-  - russellb
   - dhellmann
-  - mhrivnak
   - hardys
+  - honza
+  - mhrivnak
+  - russellb
+  - stbenjam
   - zaneb
+  - russellb
```

Comment:
you've got a duplicate trailing russellb here which needs removing

Reply:
Fixed